feature-libs/product/configurators/rulebased/cpq/cpq-configurator-normalizer.ts

```diff
@@ -0,0 +1,195 @@
+import { Injectable } from '@angular/core';
+import { Converter } from '@spartacus/core';
+import { Configurator } from './../core/model/configurator.model';
+import { Cpq } from './cpq.models';
+
+@Injectable({ providedIn: 'root' })
```

Comment:
Must not be provided in root! Is explicitly provided in rest module

Reply:
done